### ISSUE TITLE
Track habits multiple times a day

### ISSUE THREAD

**Issue by n1trux** [iSoron/uhabits#373]
It would be nice to have a way of tracking a habit which you have to keep up multiple times a day.

For example, I drink way too little. So I should fill my bottle at least 3 times a day. Would be nice if one could just have an entry "3 times a day" and have multiple checkmarks or a counter for each day.

**Comment by agepoor** [comment]
Agreed. I think nestable 'subhabits' could be a nice solution here. For instance: The main habit will be completed if all subhabits are checked.

**Comment by trymeouteh** [comment]
I agree. To choose either a checkmark or number input would be handy. That way for once a day habits you can have a checkmark but for multiple times a day you can use numbers instead of checkmarks.

A way to input the numbers via touch screen is to double tap to add 1 or to hold it to minus 1. If you have it set to Toggle With Short Press you can tap to add 1 and hold it to minus 1.

**Comment by matkoniecz** [comment]
Sounds like duplicate of #121
